server/restivus-swagger.js

```diff
@@ -4,18 +4,22 @@ const ParsedURL = Npm.require('url-parse');
 const URL = new ParsedURL(Meteor.absoluteUrl());
 
 // Add swagger route and generate valid swagger.json
-Restivus.prototype.addSwagger = function(path) {
-  // Call add Route
+Restivus.prototype.addSwagger = function(swaggerPath) {
+  // Set constants
   const restivus = this;
   const config = restivus._config;
-  restivus.addRoute(path, {authRequired: false}, {
+  const swagger = restivus.swagger;
+
+  // Call add Route
+  restivus.addRoute(swaggerPath, {authRequired: false}, {
     get: function () {
-      // Check if swagger meta configuration exists
-      if(config.swagger) {
+      // Check if swagger configuration exists
+      if(swagger !== undefined &&
+        swagger.meta !== undefined) {
         // Initialize doc object
```

Comment:
Perhaps describing the purpose of the `doc` object would be useful as a comment.